app/templates/events/view/tickets.hbs

```diff
@@ -1,15 +1,17 @@
-<div class="row">
-  <div class="{{if device.isMobile 'sixteen' 'three'}} wide column">
-    <div class="ui vertical fluid pointing menu">
-      <a class="active item">{{t 'Overview'}}</a>
-      <a class="item">{{t 'Orders'}}</a>
-      <a class="item">{{t 'Attendees'}}</a>
-      <a class="item">{{t 'Discount Codes'}}</a>
-      <a class="item">{{t 'Access Codes'}}</a>
-      <a class="item">{{t 'Add order'}}</a>
+<div class="ui grid">
```

Comment:
@anu-007 Don't make any changes to this file, this will overwrite what was fixed in #135